index.html

```diff
@@ -0,0 +1,207 @@
+        </aside>
+      </div>
+    </header>
+    <!--section second with specifications of product-->
+    <section id="features" class="specifications__container container">
+      <div class="specifications__top block">
+        <h1 class="specifications__header light">FEATURES</h1>
+      </div>
+      <div class="specifications__table block">
+        <section class="table">
+          <div class="specifications__table--background">
+            <img
+              src="/images/responsive.svg"
+              class="specifications__table--img"
+            />
+          </div>
+          <h1 class="table__title bold">FULLY RESPONSIVE</h1>
+          <h2 class="table__text light">
+            Looks amazing on any device: smartphone, tablet, laptop and desktop.
+          </h2>
+        </section>
+        <section class="table">
+          <div class="specifications__table--background">
+            <img
+              src="images/customizable.svg"
+              class="specifications__table--img"
+            />
+          </div>
+          <h1 class="table__title bold">CUSTOMIZABLE</h1>
+          <h2 class="table__text light">
+            Change the colors, pictures or any of the sections to suit your
+            needs.
+          </h2>
+        </section>
+        <section class="table">
+          <div class="specifications__table--background">
+            <img src="images/design.svg" class="specifications__table--img" />
+          </div>
+          <h1 class="table__title bold">SLICK AND BEAUTIFUL DESIGN</h1>
+          <h2 class="table__text light">
+            Trendy and fresh design, fits any website.
+          </h2>
+        </section>
+      </div>
+    </section>
+    <!--section collaborate logo brand-->
+    <section class="collaborate__container container">
+      <div class="collaborate__logo block">
+        <img src="images/logos.png" />
+      </div>
+    </section>
+    <!--section article-->
+    <section class="article">
+      <article class="article__one">
+        <img src="images/imac.png" class="article__one--img" />
+      </article>
+      <article class="article__second">
+        <h1 class="article__second_title light">SIMPLE AND BEAUTIFUL</h1>
+        <h2 class="article__second_text">
+          Use Nova theme for your next web project. It is completely
+          customizable so you can Change any of the sections to fit your needs.
+          Nova Theme is Free for kind of use, personal and commercial. Have fun
+          and good luck!
+        </h2>
+      </article>
+    </section>
+    <!--section price-->
+    <section id="pricing" class="price__container container">
+      <div class="price__main_bar block">
+        <h1 class="main__bar_title bold">CHOOSE YOUR PRICING PLAN</h1>
+        <h2 class="main__bar_text light">
+          Pick any of our super affordable pricing plans
+        </h2>
+      </div>
+      <div class="price__list block">
+        <div class="list__first">
```

Comment:
Zobacz sama nazwa klasy wzkzuje, żę to lista to może każdy element to li zamiast div, a ich rodzicem zrobić ul zamiast div? PS. Tak można mieć listę w liście ;)